docs/appendices/release-notes/5.7.3.rst

```diff
@@ -69,3 +69,11 @@ Fixes
 - Changed :ref:`scalar-pg_get_function_result` functions to be registered under
   :ref:`postgres-pg_catalog` schema, in order to be compatible with PostgreSQL
   behaviour.
+
+- Fixed an issue leading to incorrect results when joining tables on a boolean
+  expression, containing ``=`` operator and multiple relations as a part of the
```

Comment:
Rephrasing suggestion:

```
when joining tables with a ``=`` and on either side (or both) of the equality there is another ``=`` operator....

Reply:
Thx, applied. BTW, I kept 'multiple relations involved" as issue is in combination of  `=` and many relations

Reply:
thx!